app/containers/TracksContainer/index.js

```diff
@@ -1,5 +1,159 @@
-import React from 'react';
+import React, { memo, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { injectSaga } from 'redux-injectors';
+import { createStructuredSelector } from 'reselect';
+import styled from 'styled-components';
+import { tracksContainerCreators } from './reducer';
+import tracksContainerSaga from './saga';
+import { selectTrackName, selectTracksContainer, selectTracksData } from './selectors';
+import { debounce, get, isEmpty } from 'lodash';
```

Comment:
all the library imports should go to the top

Reply:
Updated the changes